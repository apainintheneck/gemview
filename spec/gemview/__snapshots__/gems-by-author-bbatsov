[
  {
    "name": "rubocop",
    "downloads": 480421146,
    "version": "1.69.1",
    "version_created_at": "2024-12-03 09:03:08 UTC",
    "authors": "Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama",
    "info": "RuboCop is a Ruby code style checking and code formatting tool.\nIt aims to enforce the community-driven Ruby Style Guide.\n",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop",
    "homepage_uri": "https://rubocop.org/",
    "source_code_uri": "https://github.com/rubocop/rubocop/",
    "changelog_uri": "https://github.com/rubocop/rubocop/releases/tag/v1.69.1",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "json",
          "requirements": "~> 2.3"
        },
        {
          "name": "language_server-protocol",
          "requirements": ">= 3.17.0"
        },
        {
          "name": "parallel",
          "requirements": "~> 1.10"
        },
        {
          "name": "parser",
          "requirements": ">= 3.3.0.2"
        },
        {
          "name": "rainbow",
          "requirements": ">= 2.2.2, < 4.0"
        },
        {
          "name": "regexp_parser",
          "requirements": ">= 2.9.3, < 3.0"
        },
        {
          "name": "rubocop-ast",
          "requirements": ">= 1.36.2, < 2.0"
        },
        {
          "name": "ruby-progressbar",
          "requirements": "~> 1.7"
        },
        {
          "name": "unicode-display_width",
          "requirements": ">= 2.4.0, < 4.0"
        }
      ]
    }
  },
  {
    "name": "guard-rubocop",
    "downloads": 14990046,
    "version": "1.5.0",
    "version_created_at": "2021-08-28 03:52:42 UTC",
    "authors": "Yuji Nakayama",
    "info": "Guard::RuboCop automatically checks Ruby code style with RuboCop when files are modified.",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/guard-rubocop",
    "homepage_uri": "https://github.com/rubocop/guard-rubocop",
    "source_code_uri": null,
    "changelog_uri": null,
    "dependencies": {
      "development": [
        {
          "name": "bundler",
          "requirements": ">= 0"
        },
        {
          "name": "guard-rspec",
          "requirements": ">= 4.2.3, < 5.0"
        },
        {
          "name": "launchy",
          "requirements": "~> 2.4"
        },
        {
          "name": "rake",
          "requirements": ">= 12.0"
        },
        {
          "name": "rspec",
          "requirements": "~> 3.0"
        },
        {
          "name": "simplecov",
          "requirements": "~> 0.7"
        }
      ],
      "runtime": [
        {
          "name": "guard",
          "requirements": "~> 2.0"
        },
        {
          "name": "rubocop",
          "requirements": "< 2.0"
        }
      ]
    }
  },
  {
    "name": "powerpack",
    "downloads": 96666755,
    "version": "0.1.3",
    "version_created_at": "2020-11-16 08:43:40 UTC",
    "authors": "Bozhidar Batsov",
    "info": "A few useful extensions to core Ruby classes.",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/powerpack",
    "homepage_uri": "https://github.com/bbatsov/powerpack",
    "source_code_uri": null,
    "changelog_uri": null,
    "dependencies": {
      "development": [
        {
          "name": "bundler",
          "requirements": ">= 1.3, < 3.0"
        },
        {
          "name": "rake",
          "requirements": ">= 0"
        },
        {
          "name": "rspec",
          "requirements": ">= 0"
        },
        {
          "name": "yard",
          "requirements": "~> 0.9"
        }
      ],
      "runtime": []
    }
  },
  {
    "name": "webpack-assets",
    "downloads": 830034,
    "version": "0.3.5",
    "version_created_at": "2019-03-29 13:10:54 UTC",
    "authors": "Sergey Nartimov",
    "info": "Webpack Assets for Rails.",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/webpack-assets",
    "homepage_uri": "https://github.com/toptal/webpack-assets",
    "source_code_uri": null,
    "changelog_uri": null,
    "dependencies": {
      "development": [
        {
          "name": "bundler",
          "requirements": "~> 1.10"
        },
        {
          "name": "rake",
          "requirements": "~> 10.0"
        },
        {
          "name": "rspec",
          "requirements": ">= 0"
        },
        {
          "name": "rspec-rails",
          "requirements": ">= 0"
        }
      ],
      "runtime": [
        {
          "name": "actionpack",
          "requirements": ">= 4.2.4"
        }
      ]
    }
  },
  {
    "name": "rubocop-sequel",
    "downloads": 2204854,
    "version": "0.3.7",
    "version_created_at": "2024-11-05 01:33:48 UTC",
    "authors": "TimothÃ©e Peignier",
    "info": "Code style checking for Sequel",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-sequel",
    "homepage_uri": "https://github.com/rubocop/rubocop-sequel",
    "source_code_uri": null,
    "changelog_uri": null,
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "rubocop",
          "requirements": "~> 1.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-rails",
    "downloads": 178716517,
    "version": "2.27.0",
    "version_created_at": "2024-10-26 08:16:39 UTC",
    "authors": "Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama",
    "info": "Automatic Rails code style checking tool.\nA RuboCop extension focused on enforcing Rails best practices and coding conventions.\n",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-rails",
    "homepage_uri": "https://docs.rubocop.org/rubocop-rails/",
    "source_code_uri": "https://github.com/rubocop/rubocop-rails/",
    "changelog_uri": "https://github.com/rubocop/rubocop-rails/blob/master/CHANGELOG.md",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "activesupport",
          "requirements": ">= 4.2.0"
        },
        {
          "name": "rack",
          "requirements": ">= 1.1"
        },
        {
          "name": "rubocop",
          "requirements": ">= 1.52.0, < 2.0"
        },
        {
          "name": "rubocop-ast",
          "requirements": ">= 1.31.1, < 2.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-performance",
    "downloads": 183053121,
    "version": "1.23.0",
    "version_created_at": "2024-11-14 18:44:37 UTC",
    "authors": "Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama",
    "info": "A collection of RuboCop cops to check for performance optimizations\nin Ruby code.\n",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-performance",
    "homepage_uri": "https://docs.rubocop.org/rubocop-performance/",
    "source_code_uri": "https://github.com/rubocop/rubocop-performance/",
    "changelog_uri": "https://github.com/rubocop/rubocop-performance/blob/master/CHANGELOG.md",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "rubocop",
          "requirements": ">= 1.48.1, < 2.0"
        },
        {
          "name": "rubocop-ast",
          "requirements": ">= 1.31.1, < 2.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-minitest",
    "downloads": 19826323,
    "version": "0.36.0",
    "version_created_at": "2024-08-31 09:02:16 UTC",
    "authors": "Bozhidar Batsov, Jonas Arvidsson, Koichi ITO",
    "info": "Automatic Minitest code style checking tool.\nA RuboCop extension focused on enforcing Minitest best practices and coding conventions.\n",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-minitest",
    "homepage_uri": "https://docs.rubocop.org/rubocop-minitest/",
    "source_code_uri": "https://github.com/rubocop/rubocop-minitest",
    "changelog_uri": "https://github.com/rubocop/rubocop-minitest/blob/master/CHANGELOG.md",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "rubocop",
          "requirements": ">= 1.61, < 2.0"
        },
        {
          "name": "rubocop-ast",
          "requirements": ">= 1.31.1, < 2.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-rake",
    "downloads": 18633605,
    "version": "0.6.0",
    "version_created_at": "2021-06-28 16:40:52 UTC",
    "authors": "Masataka Pocke Kuwabara",
    "info": "A RuboCop plugin for Rake",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-rake",
    "homepage_uri": "https://github.com/rubocop/rubocop-rake",
    "source_code_uri": "https://github.com/rubocop/rubocop-rake",
    "changelog_uri": "https://github.com/rubocop/rubocop-rake/blob/master/CHANGELOG.md",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "rubocop",
          "requirements": "~> 1.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-extension-generator",
    "downloads": 13910,
    "version": "0.5.2",
    "version_created_at": "2023-02-04 09:16:21 UTC",
    "authors": "Masataka Pocke Kuwabara",
    "info": "A generator of RuboCop's custom cops gem",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-extension-generator",
    "homepage_uri": "https://github.com/rubocop/rubocop-extension-generator",
    "source_code_uri": "https://github.com/rubocop/rubocop-extension-generator",
    "changelog_uri": null,
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "activesupport",
          "requirements": ">= 0"
        },
        {
          "name": "bundler",
          "requirements": ">= 0"
        },
        {
          "name": "rubocop",
          "requirements": ">= 1.22.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-rubycw",
    "downloads": 985601,
    "version": "0.1.6",
    "version_created_at": "2021-05-23 07:40:34 UTC",
    "authors": "Masataka Pocke Kuwabara",
    "info": "Integrate RuboCop and ruby -cw",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-rubycw",
    "homepage_uri": "https://github.com/rubocop/rubocop-rubycw",
    "source_code_uri": "https://github.com/rubocop/rubocop-rubycw",
    "changelog_uri": null,
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "rubocop",
          "requirements": "~> 1.0"
        }
      ]
    }
  },
  {
    "name": "rubocop-ast",
    "downloads": 316381524,
    "version": "1.36.2",
    "version_created_at": "2024-11-27 15:27:09 UTC",
    "authors": "Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama",
    "info": "    RuboCop's Node and NodePattern classes.\n",
    "licenses": [
      "MIT"
    ],
    "project_uri": "https://rubygems.org/gems/rubocop-ast",
    "homepage_uri": "https://www.rubocop.org/",
    "source_code_uri": "https://github.com/rubocop/rubocop-ast/",
    "changelog_uri": "https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md",
    "dependencies": {
      "development": [],
      "runtime": [
        {
          "name": "parser",
          "requirements": ">= 3.3.1.0"
        }
      ]
    }
  }
]