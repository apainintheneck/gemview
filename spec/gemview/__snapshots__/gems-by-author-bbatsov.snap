---
- !ruby/object:Gemview::Gem
  name: rubocop
  downloads: 480421146
  version: 1.69.1
  version_downloads: 279434
  version_created_at: 2024-12-03 09:03:08.778000000 Z
  authors: Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama
  info: |
    RuboCop is a Ruby code style checking and code formatting tool.
    It aims to enforce the community-driven Ruby Style Guide.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop
  homepage_uri: https://rubocop.org/
  source_code_uri: https://github.com/rubocop/rubocop/
  changelog_uri: https://github.com/rubocop/rubocop/releases/tag/v1.69.1
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: json
    requirements: "~> 2.3"
  - !ruby/object:Gemview::Gem::Dependency
    name: language_server-protocol
    requirements: ">= 3.17.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: parallel
    requirements: "~> 1.10"
  - !ruby/object:Gemview::Gem::Dependency
    name: parser
    requirements: ">= 3.3.0.2"
  - !ruby/object:Gemview::Gem::Dependency
    name: rainbow
    requirements: ">= 2.2.2, < 4.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: regexp_parser
    requirements: ">= 2.9.3, < 3.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop-ast
    requirements: ">= 1.36.2, < 2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: ruby-progressbar
    requirements: "~> 1.7"
  - !ruby/object:Gemview::Gem::Dependency
    name: unicode-display_width
    requirements: ">= 2.4.0, < 4.0"
- !ruby/object:Gemview::Gem
  name: guard-rubocop
  downloads: 14990046
  version: 1.5.0
  version_downloads: 5721654
  version_created_at: 2021-08-28 03:52:42.437000000 Z
  authors: Yuji Nakayama
  info: Guard::RuboCop automatically checks Ruby code style with RuboCop when files
    are modified.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/guard-rubocop
  homepage_uri: https://github.com/rubocop/guard-rubocop
  source_code_uri:
  changelog_uri:
  development_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: bundler
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: guard-rspec
    requirements: ">= 4.2.3, < 5.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: launchy
    requirements: "~> 2.4"
  - !ruby/object:Gemview::Gem::Dependency
    name: rake
    requirements: ">= 12.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rspec
    requirements: "~> 3.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: simplecov
    requirements: "~> 0.7"
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: guard
    requirements: "~> 2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: "< 2.0"
- !ruby/object:Gemview::Gem
  name: powerpack
  downloads: 96666755
  version: 0.1.3
  version_downloads: 9595185
  version_created_at: 2020-11-16 08:43:40.093000000 Z
  authors: Bozhidar Batsov
  info: A few useful extensions to core Ruby classes.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/powerpack
  homepage_uri: https://github.com/bbatsov/powerpack
  source_code_uri:
  changelog_uri:
  development_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: bundler
    requirements: ">= 1.3, < 3.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rake
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rspec
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: yard
    requirements: "~> 0.9"
  runtime_dependencies: []
- !ruby/object:Gemview::Gem
  name: webpack-assets
  downloads: 830034
  version: 0.3.5
  version_downloads: 184778
  version_created_at: 2019-03-29 13:10:54.151000000 Z
  authors: Sergey Nartimov
  info: Webpack Assets for Rails.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/webpack-assets
  homepage_uri: https://github.com/toptal/webpack-assets
  source_code_uri:
  changelog_uri:
  development_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: bundler
    requirements: "~> 1.10"
  - !ruby/object:Gemview::Gem::Dependency
    name: rake
    requirements: "~> 10.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rspec
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rspec-rails
    requirements: ">= 0"
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: actionpack
    requirements: ">= 4.2.4"
- !ruby/object:Gemview::Gem
  name: rubocop-sequel
  downloads: 2204854
  version: 0.3.7
  version_downloads: 36176
  version_created_at: 2024-11-05 01:33:48.206000000 Z
  authors: TimothÃ©e Peignier
  info: Code style checking for Sequel
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-sequel
  homepage_uri: https://github.com/rubocop/rubocop-sequel
  source_code_uri:
  changelog_uri:
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: "~> 1.0"
- !ruby/object:Gemview::Gem
  name: rubocop-rails
  downloads: 178716517
  version: 2.27.0
  version_downloads: 2211846
  version_created_at: 2024-10-26 08:16:39.168000000 Z
  authors: Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama
  info: |
    Automatic Rails code style checking tool.
    A RuboCop extension focused on enforcing Rails best practices and coding conventions.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-rails
  homepage_uri: https://docs.rubocop.org/rubocop-rails/
  source_code_uri: https://github.com/rubocop/rubocop-rails/
  changelog_uri: https://github.com/rubocop/rubocop-rails/blob/master/CHANGELOG.md
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: activesupport
    requirements: ">= 4.2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rack
    requirements: ">= 1.1"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: ">= 1.52.0, < 2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop-ast
    requirements: ">= 1.31.1, < 2.0"
- !ruby/object:Gemview::Gem
  name: rubocop-performance
  downloads: 183053121
  version: 1.23.0
  version_downloads: 791266
  version_created_at: 2024-11-14 18:44:37.733000000 Z
  authors: Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama
  info: |
    A collection of RuboCop cops to check for performance optimizations
    in Ruby code.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-performance
  homepage_uri: https://docs.rubocop.org/rubocop-performance/
  source_code_uri: https://github.com/rubocop/rubocop-performance/
  changelog_uri: https://github.com/rubocop/rubocop-performance/blob/master/CHANGELOG.md
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: ">= 1.48.1, < 2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop-ast
    requirements: ">= 1.31.1, < 2.0"
- !ruby/object:Gemview::Gem
  name: rubocop-minitest
  downloads: 19826323
  version: 0.36.0
  version_downloads: 1676797
  version_created_at: 2024-08-31 09:02:16.815000000 Z
  authors: Bozhidar Batsov, Jonas Arvidsson, Koichi ITO
  info: |
    Automatic Minitest code style checking tool.
    A RuboCop extension focused on enforcing Minitest best practices and coding conventions.
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-minitest
  homepage_uri: https://docs.rubocop.org/rubocop-minitest/
  source_code_uri: https://github.com/rubocop/rubocop-minitest
  changelog_uri: https://github.com/rubocop/rubocop-minitest/blob/master/CHANGELOG.md
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: ">= 1.61, < 2.0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop-ast
    requirements: ">= 1.31.1, < 2.0"
- !ruby/object:Gemview::Gem
  name: rubocop-rake
  downloads: 18633605
  version: 0.6.0
  version_downloads: 15753508
  version_created_at: 2021-06-28 16:40:52.850000000 Z
  authors: Masataka Pocke Kuwabara
  info: A RuboCop plugin for Rake
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-rake
  homepage_uri: https://github.com/rubocop/rubocop-rake
  source_code_uri: https://github.com/rubocop/rubocop-rake
  changelog_uri: https://github.com/rubocop/rubocop-rake/blob/master/CHANGELOG.md
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: "~> 1.0"
- !ruby/object:Gemview::Gem
  name: rubocop-extension-generator
  downloads: 13910
  version: 0.5.2
  version_downloads: 977
  version_created_at: 2023-02-04 09:16:21.369000000 Z
  authors: Masataka Pocke Kuwabara
  info: A generator of RuboCop's custom cops gem
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-extension-generator
  homepage_uri: https://github.com/rubocop/rubocop-extension-generator
  source_code_uri: https://github.com/rubocop/rubocop-extension-generator
  changelog_uri:
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: activesupport
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: bundler
    requirements: ">= 0"
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: ">= 1.22.0"
- !ruby/object:Gemview::Gem
  name: rubocop-rubycw
  downloads: 985601
  version: 0.1.6
  version_downloads: 912052
  version_created_at: 2021-05-23 07:40:34.571000000 Z
  authors: Masataka Pocke Kuwabara
  info: Integrate RuboCop and ruby -cw
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-rubycw
  homepage_uri: https://github.com/rubocop/rubocop-rubycw
  source_code_uri: https://github.com/rubocop/rubocop-rubycw
  changelog_uri:
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: rubocop
    requirements: "~> 1.0"
- !ruby/object:Gemview::Gem
  name: rubocop-ast
  downloads: 316381524
  version: 1.36.2
  version_downloads: 826979
  version_created_at: 2024-11-27 15:27:09.608000000 Z
  authors: Bozhidar Batsov, Jonas Arvidsson, Yuji Nakayama
  info: "    RuboCop's Node and NodePattern classes.\n"
  licenses:
  - MIT
  project_uri: https://rubygems.org/gems/rubocop-ast
  homepage_uri: https://www.rubocop.org/
  source_code_uri: https://github.com/rubocop/rubocop-ast/
  changelog_uri: https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md
  development_dependencies: []
  runtime_dependencies:
  - !ruby/object:Gemview::Gem::Dependency
    name: parser
    requirements: ">= 3.3.1.0"
